projects/index.md

```diff
@@ -12,23 +12,25 @@ Would you like your project added to the list? We would!
 <ul id="projects-list">
     {% for post in site.posts %}
         {% if post.categories contains "projects" %}
-            <li>
-                <div class="project-tile-title">
-                    <h4>
-                        <a href="{{ site.github.url }}{{ post.url }}">{{ post.title }}</a>
-                    </h4>
-                </div>
-                {% assign num_words = 40 %}
-                {% if post.youtube_video_id %}
-                    {% assign num_words = 20 %}
-                    <a href="{{ site.github.url }}{{ post.url }}">
-                        <img class="project-tile-img" src="http://img.youtube.com/vi/{{post.youtube_video_id}}/mqdefault.jpg" />
-                    </a>
-                {% endif %}
-                <span class="project-tile-excerpt">
-                    {{ post.excerpt | truncatewords: num_words }}
-                </span>
-            </li>
+            {% unless post.title contains "Example" %}
```

Comment:
What if someone else uses the word "Example" in their title? Can we make this a more exact match?